### PR TITLE
update channel moderation doc

### PR DESCRIPTION
**Pull request by bhartono**

### CODE REVIEW

File: api-reference/beta/resources/channelmoderationsettings.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## JSON representation
+The following is a JSON representation of the resource.
+<!-- {
+  "blockType": "resource",
+  "@odata.type": "microsoft.graph.channelModerationSettings2"
+}
+-->
+``` json
+{
+  "@odata.type": "#microsoft.graph.channelModerationSettings2",
```
Comment (AkJo) [inline api-reference/beta/resources/channelmoderationsettings.md:45]: ?

File: api-reference/beta/resources/channelmoderationsettings.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+|allowNewMessageFromBots|Boolean|Indicates whether bots are allowed to post messages.|
+|allowNewMessageFromConnectors|Boolean|Indicates whether connectors are allowed to post messages.|
+|replyRestriction|[replyRestriction](../resources/enums.md#replyRestriction-values)|Indicates who all are allowed to reply to teams channel.|
+|userNewMessageRestriction|[userNewMessageRestriction](../resources/enums.md#userNewMessageRestriction-values)|Indicates who all are allowed to post messages to teams channel.|
+
+## JSON representation
+The following is a JSON representation of the resource.
+<!-- {
+  "blockType": "resource",
+  "@odata.type": "microsoft.graph.channelModerationSettings2"
```
Comment (AkJo) [inline api-reference/beta/resources/channelmoderationsettings.md:40]: ?

File: api-reference/beta/resources/channelmoderationsettings.md (hunk 1 above)
Comment (bhartono) [reply api-reference/beta/resources/channelmoderationsettings.md:45]: fixed

**Comment by bhartono** [comment]
@Lauragra please merge

**Comment by bhartono** [comment]
@Lauragra @AkJo  can we please merge this ?

File: api-reference/beta/api/channel-post.md [hunk 3]
```diff
@@ -218,14 +192,82 @@ Content-Type: application/json
...
+The following example shows the response.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "name": "create_channel_through_migration",
+  "@odata.type": "microsoft.graph.channel"
+} -->
+
 ```http
 HTTP/1.1 202 Accepted
```
Comment (Lauragra) [inline api-reference/beta/api/channel-post.md:203]: This response scenario isn't described in the Response section - do you need to add it?

File: changelog/changelog.json [hunk 4]
```diff
@@ -1,5 +1,47 @@
 {
   "changelog": [
+    {
```
Comment (Lauragra) [inline changelog/changelog.json:3]: Please edit the Teams.Core.json file instead of this one.

File: api-reference/beta/api/channel-post.md (hunk 3 above)
Comment (AkJo) [reply api-reference/beta/api/channel-post.md:204]: Added some description.

**Pull request merged**